frontend/pkg/frontend/routes.go

```diff
@@ -44,6 +44,7 @@ func (f *Frontend) routes() *MiddlewareMux {
 	metricsMiddleware := MetricsMiddleware{dbClient: f.dbClient, Emitter: f.metrics}
 
 	mux := NewMiddlewareMux(
+		MiddlewareTracing,
```

Comment:
Should we move it after the panic middleware? I assume that the panic middleware should wrap all other middlewares.

Reply:
Thats a good  question. I wasnt sure. How its setup we should complete the trace and report the `panic` and stack trace. I assume it we place it in front the trace will be gone.

Reply:
ftr: Discussed this with @simonpasquier and @dinhxuanvu.
Maybe it makes sense to register the panicMiddleware twice?
e.g.
```
MiddlewarePanic
MiddlewareTracing
metricsMiddleware.Metrics()
MiddlewareLogging
MiddlewarePanic
...
others
```